### PR TITLE
Correcting ip addresses and fixing file structure.

### PR DESCRIPTION
**Pull request by danielmgit**

### CODE REVIEW

**Comment by danielmgit** [comment]
It was inevitable.

File: conf.json [hunk 1]
```diff
@@ -29,11 +31,11 @@
 				"interfaces": [
 				   {
 						 "name": "left",
-						 "ip-address": "10.0.0.1",
+						 "ip-address": "10.0.0.255",
```
Comment (hgn) [inline conf.json]: probably 10.0.0.250? 255 is the broadcast address for /24

File: conf.json [hunk 2]
```diff
@@ -29,11 +31,11 @@
...
 					 },
 					 {
 						 "name": "right",
-						 "ip-address": "10.0.1.1",
+						 "ip-address": "10.0.1.255",
```
Comment (hgn) [inline conf.json]: Same same

**Comment by hgn** [comment]
If fixed I will merge the commits

**Pull request closed without merging**